app/views/users/_users_table.html.erb

```diff
@@ -0,0 +1,30 @@
+  <table class="table border-side">
+    <thead>
+      <tr>
+        <% if local_assigns[:sortable] %>
+          <th><!-- handle --></th>
+        <% end %>
+        <th><input type="checkbox" id="select-all-users"></th>
+        <th>Full Name</th>
+        <th>Role</th>
+        <th>Last Login</th>
+        <%= th_autosortable 'Date Created', 'string', local_assigns %>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% if users.nil? %>
+      <% else %>
+        <% users.each do |user| %>
```

Comment:
Does `users.each` without the nil guard not work here? Usually you can `each` on empty arrays, which is what users should probably be. Let me know if that's a problem and we can write a workaround. $$